lib/rspec/support/comparable_version.rb

```diff
@@ -1,46 +0,0 @@
-module RSpec
```

Comment:
Restore this please, it's used for better comparison of the 4 digit JRuby strings.